src/mavo.js

```diff
@@ -426,16 +426,30 @@ let _ = self.Mavo = $.Class(class Mavo {
 		let existing = this[role], backend, changed;
 		const attribute = "mv-" + role;
 
+		// If we override a backend via URL params,
+		// we should ignore metadata the author might pass via storage attributes.
+		// In other words, URLs should have precedence over everything static.
+		let ignoreRoleAttributes = false;
+
 		if (this.index == 1) {
 			// This app is the first one in the page, so we can override its backend
 			// via URL params such as ?storage=...
 			backend = _.Functions.url(role);
+
+			if (backend) {
+				ignoreRoleAttributes = true;
+			}
 		}
 
 		if (!backend) {
```

Comment:
Was this if statement working before, with `backend` uninitialized?

Reply:
Presumably it would always be true unless `this.index == 1`

Reply:
AFAIK this works precisely as Lea described.